package.json

```diff
@@ -23,6 +23,7 @@
     "test": "node -e \"require('grunt').cli()\" null test"
   },
   "devDependencies": {
+    "jslint": "^0.12.1",
```

Comment:
Where is the code using `jslint`? I mentioned tooling like this belongs in `devDependencies` if used, but I don't see it being used anywhere in this PR so I think it's best to remove this.

Reply:
Ok

Reply:
Done

Reply:
Is it now ready to merge??

Reply:
Please be patient; this is an open-source project and we don't always have the time to respond to PRs right away. I do check my issue queues/notifications but I can't get to things right away 🙂

Reply:
Ok. No worry.